tests/login.test.js

```diff
@@ -0,0 +1,55 @@
+import LoginPage from '../pages/login.page';
+
+describe('DigitalBank', () => {
```

Comment:
This suite is actually about testing login functionality. This name is too broad. Consider using something that refers to the fact that you are testing login stuff